src/containers/shedule/Schedule.jsx

```diff
@@ -31,7 +31,7 @@ const Schedule = ({ residentsData, currentShow }) => {
             ? Object.keys(fiveDayAirTimeSchedule).map((day, i) => {
                 const dayData = fiveDayAirTimeSchedule[day];
                 return (
-                  <>
+                  <Fragment key={`schedule-${i}`}>
                     <Name key={`day-${day}-${i}`}>
```

Comment:
Not sure why I have the key on the `<Name>` - it can probably be removed?

Reply:
I think it's nothing to do with you, it was written by me and I think I had there the key on Name until I needed to wrap it in a Fragment, and then forgot to remove. [Removed](https://github.com/RichardPK/eh-fm/pull/46/commits/40085fd38f527a84a7c4c83edd7af816fe7b9dc2) Thanks for a good spot!